docs/changelog.md

```diff
@@ -5,6 +5,22 @@ toc:
 
 # Redocly CLI changelog
 
+
+## 1.0.0-beta.127 (2023-05-?)
+
+### Features
+
+- Display an update call-to-action if a new version is available.
```

Comment:
Could we just say "inform the user if a new version is available" please. "Call to action" feels like internal marketing language (also this sentence doesn't make sense as it is)

Reply:
@adamaltman  I see this is your change, what do you think? 
```suggestion
- Inform the user if a new version is available.
```